### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by YueErro**

### CODE REVIEW

**Comment by rkojcev** [comment]
please resolve conflicts before

File: INSTALL.md [hunk 1]
```diff
@@ -18,26 +19,17 @@ For the complete MARA experiments installation, please refer first to the **ROS2
...
-ros-crystal-tf2 \
-ros-crystal-tf2-geometry-msgs \
-ros-crystal-rclcpp-action \
 ros-crystal-cv-bridge \
 ros-crystal-control-msgs \
 ros-crystal-image-transport \
 ros-crystal-gazebo-dev \
 ros-crystal-gazebo-msgs \
 ros-crystal-gazebo-plugins \
-ros-crystal-gazebo-ros \
```
Comment (nzlz) [inline INSTALL.md]: I think we need this. reset services are located there.

File: INSTALL.md [hunk 2]
```diff
@@ -149,15 +141,3 @@ cd gym-gazebo2
...
-
-```diff
--  source /opt/ros/crystal/setup.bash
-+  source ~/ros2_ws/install/setup.bash
-   source ~/ros2_mara_ws/install/setup.bash
-   source /usr/share/gazebo-9/setup.sh
-   export PYTHONPATH=$PYTHONPATH:~/ros2_mara_ws/install/lib/python3/dist-packages
-   export GAZEBO_MODEL_PATH=$GAZEBO_MODEL_PATH:~/ros2_mara_ws/src/MARA
-   export GAZEBO_PLUGIN_PATH=$GAZEBO_PLUGIN_PATH:~/ros2_mara_ws/src/MARA/mara_gazebo_plugins/build/
-```
```
Comment (nzlz) [inline INSTALL.md]: Why are you removing this? Users who installed crystal using source installation need to know this.

File: INSTALL.md (hunk 1 above)
Comment (YueErro) [reply INSTALL.md]: Tested on ubuntu:bionic docker and installed ros-crystal-desktop, ros-crystal-gazebo-ros got installed automatically

File: INSTALL.md (hunk 2 above)
Comment (YueErro) [reply INSTALL.md]: Ups, this is a mistake, thanks.

Comment (rkojcev) [reply INSTALL.md]: revert back in that case. We need to consider those users who are installing from source

**Comment by nzlz** [comment]
Approved but I still cant see where gazebo_ros is coming from through `ros-crystal-desktop`.

**Comment by YueErro** [comment]
You can add it back it will not break anything. I have only changed the things according to what i have experimented.

**Comment by nzlz** [comment]
For the next times: updates in the Instructions (readmes) should be combined with the corresponding updates in the Dockerfiles (including the CI one).

Ill make the changes related to this PR tomorrow.

**Pull request merged**